internal/workflow.go

```diff
@@ -1565,11 +1565,94 @@ const DefaultVersion Version = -1
 // CadenceChangeVersion is used as search attributes key to find workflows with specific change version.
 const CadenceChangeVersion = "CadenceChangeVersion"
 
+// GetVersionOptions contains options for GetVersion
+type GetVersionOptions struct {
```

Comment:
This naming is super confusing. `GetVersion` actually means "get version or set if empty". Here this option only change how we set version. Thus may I suggest name change to be `SetVersionOptions`?

Reply:
Decided to keep it `GetVersionOptions`, as for option structures for functions we use a naming convention - `<FuncName>Options`. Anyway, it's a internal type, so we will always have an option to rename it.